tools/err.c

```diff
@@ -0,0 +1,116 @@
+#if defined(__WIN32) || defined(_AIX)
```

Comment:
Do these files really belong into the tools directory ? 
Currently the tools directory contains tools (executables) that are used during build.

I guess these AIX specific implementations should rather go somewhere else. Maybe under `usr/lib/aix/` ? Or `usr/lib/platform/aix/` ?

I guess the AIX specific headers should then also go there. `platform.h` could go right into `usr/lib/common/` though, or into `usr/lib/platform/` if we go that way.

Reply:
I will move all the aix-specific implementation files to `usr/lib/common/aix`. However, I think the header files should be fine in their current location, because they're already in their own `aix/` folder within `usr/include`.

Reply:
For me `usr/include` is more a location for includes that get installed (although not entirely true for the header files in there for historical reasons). Includes that are not installed should rather go into `usr/lib/common` or in their own subdirectory if they belong to a component that lives in its own subdirectory.